src/nnet3/nnet-am-decodable-simple.cc

```diff
@@ -237,6 +246,23 @@ void NnetDecodableBase::DoNnetComputation(
   request.outputs.resize(1);
   request.outputs[0].Swap(&output_spec);
 
+  // add recurrent inputs to request
```

Comment:
State that the recurrent inputs being added are just dummies and we do not actually preserve the state of the network across chunks during decoding.